setup/src/main/java/io/jenkins/jenkinsfile/runner/JenkinsLauncher.java

```diff
@@ -107,6 +109,15 @@ protected ServletContext createWebServer() throws Exception {
         return context.getServletContext();
     }
 
+    private String validateHttpPath(String httpPath) throws IllegalArgumentException {
+        if (StringUtils.endsWith(httpPath, "/")) {
+            throw new IllegalArgumentException("httpPath must not end with a trailing /");
+        } else if ("".equals(httpPath)) {
```

Comment:
```suggestion
        } else if (StringUtils.isBlank(httpPath)) {
```

Reply:
Not in this case. `null` is a valid input (what you get when the user does not set `--httpPath`)